Documentation/a-first-example.rst

```diff
@@ -11,39 +11,50 @@ in this tutorial. We assume that you have successfully compiled SeisSol.
 Setup
 -----
 
--  Download the `parameter file and additional setup
-   files <https://github.com/SeisSol/Examples/tree/master/tpv33>`__ and
-   put them in your launch directory, hereafter named
-   ``launch_SeisSol``.
--  Download the mesh `binary
-   file <https://syncandshare.lrz.de/getlink/fi72mQiszp6vSs7qN8tdZJf9/tpv33_gmsh>`__
-   and the associated `xml
-   file <https://syncandshare.lrz.de/getlink/fiEi52Xiwwqkf2sNpTrCHjhw/tpv33_gmsh.xdmf>`__
-   and store them in ``launch_SeisSol``.
-   **Optional:** For performance reasons, we suggest that you store the
+*  Clone our examples repository: https://github.com/SeisSol/Examples/.
+
+*  Navigate to the folder ``tpv33``.
+
+*  To create the mesh, execute ``./generating_the_mesh.sh``. To do so,
```

Comment:
When this page was created, it was argued (e.g. Alice) that it would be better if the mesh was already generated and ready to download. Could we (=you :-)) put the mesh on zenodo and remove the details of its generation?

Reply:
I suppose it would be easier for the users, up to you then, as it is extra work that you ask for ;-)

Reply:
Not gonna do it. Just a good idea :D